template/secret.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: my-secret
+  namespace: myproject
+type: Opaque
+data:
+  OS_USERNAME: eW91cnBvd2VydmN1c2VybmFtZQo=
```

Comment:
Can we remove the values of OS_USERNAME and OS_PASSWORD

Reply:
Have updated the contents now.